remote-storage-managers/s3/src/main/java/org/apache/kafka/rsm/s3/S3RemoteStorageManagerConfig.java

```diff
@@ -0,0 +1,146 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.kafka.rsm.s3;
+
+import java.util.Map;
+
+import org.apache.kafka.common.KafkaException;
+import org.apache.kafka.common.config.AbstractConfig;
+import org.apache.kafka.common.config.ConfigDef;
+import org.apache.kafka.common.config.ConfigException;
+
+import com.amazonaws.auth.AWSCredentialsProvider;
+import com.amazonaws.regions.Regions;
+
+/**
+ * A configuration for {@link S3RemoteStorageManager}.
+ */
+public class S3RemoteStorageManagerConfig extends AbstractConfig {
+    public static final String S3_BUCKET_NAME_CONFIG = "s3.bucket.name";
```

Comment:
Hello @satishd  I guess you miss the prefix  REMOTE_STORAGE_MANAGER_CONFIG_PREFIX.

Reply:
Maybe it's worth doing `config.originalsWithPrefix(REMOTE_STORAGE_MANAGER_CONFIG_PREFIX)` instead of adding the prefix manually to all configurations. I'll look into this.

Reply:
Fixed now